community/sway/etc/sway/config.d/99-autostart-applications.conf

```
@@ -18,4 +18,5 @@ exec_always {
     # restart kanshi https://github.com/emersion/kanshi/issues/43#issuecomment-531679213
     '[ -x "$(command -v kanshi)" ] && pkill kanshi; exec kanshi'
     '[ -x "$(command -v sworkstyle)" ] && pkill sworkstyle; sworkstyle &> /tmp/sworkstyle.log'
+    "pkill -0 playerctl; playerctl --follow metadata | pkill -RTMIN+5 waybar"
```

```suggestion
'[-x "$(command -v playerctl)" ] && pkill -0 playerctl; playerctl --follow metadata | pkill -RTMIN+5 waybar' 
```
Just to be more standard and stuff :D